colcon_cmake/task/cmake/build.py

```diff
@@ -101,15 +101,15 @@ def add_arguments(self, *, parser):  # noqa: D102
 
         rc = await self._build(
             args, env, additional_targets=additional_targets)
-        if rc and rc.returncode:
-            return rc.returncode
+        if rc:
+            return rc
```

Comment:
Is this supposed to be `rc.returncode` to keep the return type consistent?

Reply:
This change goes together with your other comment. The `_build` method now returns just the integer return code (instead of a `CompletedProcess` before).